src/f32/vec4_mask.rs

```diff
@@ -29,6 +32,34 @@ impl Default for Vec4Mask {
     }
 }
 
+#[cfg(vec3sse2)]
```

Comment:
Should be `#[cfg(vec4sse2)]` here. This is probably why the CI build is failing.